src/components/CreateAnalysis/ImageSelectionBox.tsx

```diff
@@ -33,8 +33,8 @@ const ImageSelectionBox: React.FC<ImageSelectionBoxProps> = ({ img }) => {
           <img src={image_placeholder} alt="selected" width={60} height={60} />
           <div className="margin_left">
             <p className="color_grey">{StudyDescription}</p>
-            <p className="color_grey">Filename{fname.split('/')[3]}</p>
-            <p className="color_grey">{formatDate(creation_date)}</p>
+            <p className="color_grey">{"Filename: " + fname.split('/')[3]}</p>
```

Comment:
Would it be better to move this commit to another PR? it's not wholly related, but I made this change in the process of walking through the process of solving this issue.

Reply:
Before:
![Screenshot from 2021-05-13 17-40-19](https://user-images.githubusercontent.com/53355975/118192039-8b565000-b413-11eb-886d-f7a807c0e62c.png)
After:
![Screenshot from 2021-05-13 17-36-32](https://user-images.githubusercontent.com/53355975/118192070-94472180-b413-11eb-95ce-60da6e802a3b.png)